command/certificate/create.go

```diff
@@ -263,21 +264,32 @@ $ step certificate create --template root.tpl \
 '''
 
 Create an intermediate certificate using the previous root. This intermediate
-will be able to sign also new intermediate certificates:
+will be able to sign also new intermediate certificates. We will also make the
+subject configurable using the **--set** and **--set-file** flags.
 '''
 $ cat intermediate.tpl
 {
 	"subject": {
-		"commonName": "Acme Corporation Intermediate CA"
+		"country": {{ toJson .Insecure.User.country }},
+		"organization": {{ toJson .Insecure.User.organization }},
+		"organizationalUnit": {{ toJson .Insecure.User.organizationUnit }},
+		"commonName": {{toJson .Subject.CommonName }}
 	},
 	"keyUsage": ["certSign", "crlSign"],
 	"basicConstraints": {
 		"isCA": true,
 		"maxPathLen": 1
 	}
 }
+$ cat organization.json
+{
+	"country": "US",
+	"organization": "Acme Corporation",
+	"organizationUnit": "HQ"
+}
```

Comment:
You're setting `organizationUnit` in the file and on the cmd line.

Reply:
Yes, on purpose, the --set flag overwrites the one in the file